src/stories/Header.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Navbar from '../components/navbar'
+import gLogo from './assets/gLogo.svg'
+
+const ITEMS = [
```

Comment:
Since its a static page, curious if I want it all frontend or provide this from the server aspect when running the web app.